public/index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/modules.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/icon.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.min.js"></script>
+    <script src="vendors/scripts/page.js"></script>
+    <title>You Are Hungry!</title>
+    <script id="search-results" type="text/x-handlebars-template">
+      <article class="results-cards">
+        <a href="/{{id}}"><h1 class="css-recipe-header">{{recipeName}}</h1></a>
+        <section>by {{sourceDisplayName}}</section>
+        <a href="/{{id}}"><img src="{{smallImageUrls}}" target="_blank" class="css-full-img"></a>
+      </article>
+    </script>
+    <script id="recipe-selection" type="text/x-handlebars-template">
+      <article class="results-cards">
+        <section id="recipe-name"><h1 class="css-recipe-header">{{name}}</h1></section>
+        <section id="sourceDisplayName"><a href="{{source.sourceRecipeUrl}}">{{source.sourceDisplayName}}</a></section>
+        <section id="sourceSiteUrl"></section>
+        <section id="rating">Rating: <span>{{rating}}</span> out of 5</section><br>
+        <section><img id="splash-image" src="null" class="css-full-img"></section>
+        <section id="ingredients-results">
+          <button id="ingredients-button" type="button" class="css-flex-small-ingredients-button">Add Ingredients to Shopping List</button>
+          <ul id="ingredients-list"></ul>
+        </section>
+        <a href="{{source.sourceRecipeUrl}}" target="_blank"><button class="css-directions-button">READ INSTRUCTIONS</button></a>
+      </article>
+    </script>
+    <script id="listItem-template" type="text/x-handlebars-template">
+      <li>
+        <input type="checkbox">
+        <label>{{item}}</label>
+        <button class="delete">Delete</button>
+      </li>
+    </script>
+  </head>
+  <img src="img/hungry.png" class="css-home-logo">
+  <a href="/login" class="css-home-login-link">Log In</a>
+  <body>
+    <div id="pattern" class="pattern">
+      <div class="wrap" id="wrap">
+        <header id="mainNav">
+          <a href="" class="menu-link"><i class="icon-spoon-knife"></i></a>
+          <nav id="menu" role="navigation">
```

Comment:
i think it's redundant to include role attributes for elements that already have descriptive element tags like <nav>